into/backends/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def sc():
+    pyspark = pytest.importorskip('pyspark')
+    return pyspark.SparkContext('local', 'into-test')
+
+
+@pytest.fixture(scope='session')
+def sqlctx(sc):
+    sparksql = pytest.importorskip('pyspark.sql')
+    return sparksql.SQLContext(sc)
```

Comment:
I'd like to post-pone doing that because I'd rather not add any extra classes right now. If you think this is critical, though, I'll do it.